safe/impact_functions/generic/classified_polygon_people/impact_function.py

```diff
@@ -168,7 +200,7 @@ def run(self):
         extra_keywords = {
             'impact_summary': impact_summary,
             'target_field': self.target_field,
-            'map_title': tr('Affected Areas'),
+            'map_title': tr('Affected Populations'),
```

Comment:
Probably need @Charlotte-Morgan 's input here, but 'people' makes more sense to me than 'Populations'